website/static/js/pages/register_1-page.js

```diff
@@ -65,6 +65,18 @@ $(document).ready(function() {
 
     $('#registration_template form').on('submit', function() {
 
+        // If embargo is requested, verify its end date, and inform user if date is out of range
+        if (registrationViewModel.embargoAddon.requestingEmbargo()) {
+            if (!registrationViewModel.embargoAddon.isEmbargoEndDateValid()) {
+                registrationViewModel.continueText('');
+                $osf.growl(
+                    'Invalid embargo end date',
+                    'Please double check the date you have chosen is within the parameters specified.',
```

Comment:
See https://github.com/CenterForOpenScience/osf.io/pull/2647#issuecomment-104135537 for suggested change.